### PR TITLE
Azure: Fix lowercase  hostname issue when downloading logs form VM

### PR DESCRIPTION
**Pull request by asofron**
Issue: #129
Waagent on RHEL  based distribution will change hostname to lowercase 

https://github.com/Azure/WALinuxAgent/issues/686

### CODE REVIEW

File: Libraries/LISAV2-Framework.psm1 [hunk 1]
```diff
@@ -1159,6 +1159,12 @@ Function Do-TestCleanUp($CurrentTestResult, $testName, $DeployedServices, $Resou
 						$vmData = $allVMData
 					}
 					$FilesToDownload = "$($vmData.RoleName)-*.txt"
+					#Check for lowercase filenames (On Azure waagent changes hostname to lowercase)
+					$FilesToDownloadToLower = $FilesToDownload.ToLower()
+					$CheckForLowerCase = Run-LinuxCmd -username $user -password $password -ip $vmData.PublicIP -port $vmData.SSHPort -command "ls|grep $($vmData.RoleName.ToLower())" -ignoreLinuxExitCode -runAsSudo
```
[the remark this answers is not in the document]
Comment (asofron) [reply Libraries/LISAV2-Framework.psm1]: Checking that logs file with lowercase name exist

File: Libraries/LISAV2-Framework.psm1 [hunk 2]
```diff
@@ -1159,6 +1159,12 @@ Function Do-TestCleanUp($CurrentTestResult, $testName, $DeployedServices, $Resou
...
+					if (($TestPlatform -eq "Azure") -and (-not $CheckForLowerCase)) {
+						$FilesToDownload = $FilesToDownloadToLower
```
[the remark this answers is not in the document]
Comment (chvalean) [reply Libraries/LISAV2-Framework.psm1]: This is only specific to this situation as it was logged in the issues.

**Comment by ader1990** [comment]
@asofron why not solve the cause and not the symptom?
when setting the hostname, we can have it lower case (the name of the instance should be lower case)

**Comment by LiliDeng** [comment]
@asofron Have you repro this issue? Based my testing against RHEL 7.5 and CentOS 7.5, not repro this issue. Then I see [Here](https://github.com/Azure/WALinuxAgent/pull/690), this issue for WALA is resolved.

**Comment by simonxiaoss** [comment]
In this case, we don’t need to do something in LISAv2.
Please update the WALA bug with what you found.

**Comment by asofron** [comment]
Created issue #1413 for WaLinuxAgent with relevant information

**Comment by LiliDeng** [comment]
https://github.com/Azure/WALinuxAgent/issues/1413 is marked as a bug

**Pull request closed without merging**